docker/compose-ng.sls

```diff
@@ -2,13 +2,21 @@
 {%- set tplroot = tpldir.split('/')[0] %}
 {%- from tplroot ~ "/map.jinja" import docker with context %}
 {%- from tplroot ~ "/map.jinja" import compose with context %}
+{%- from tplroot ~ "/map.jinja" import networks with context %}
 
 include:
   - docker.compose
 
+{%- for networkname in networks %}
+{{ networkname }} network:
+  docker_network.present:
+    - name: {{ networkname }}
+{%- endfor %}
+
```

Comment:
I think this should go in a separate `networks.sls` and just added to the list of `include:`.

This will allow docker.containers to benefit from that (not related but can be done in another PR).

Also, there are other kinds of docker networks and this would be easier to handle them in a separate file.

Reply:
I have now swapped it to a different file and added to networks.sls. It is indeed a good idea for future expansions!